src/test/rustdoc-ui/intra-links-ambiguity.rs

```diff
@@ -34,3 +34,7 @@ pub mod foo {
 ///
 /// Ambiguous non-implied shortcut link [`foo::bar`]. //~ERROR `foo::bar`
 pub struct Docs {}
+
+/// [true] //~ ERROR
```

Comment:
```suggestion
/// [true] //~ ERROR `true` is both a module and a builtin type
```

Reply:
I don't really understand the purpose of this error marker - specifying the message here and in the `.stderr` seems like a duplication of effort for me. Granted, this is a bit of double-check, but I'm not convinced it's necessary.

Reply:
It's not necessary for the _testsuite_; it's helpful for the reviewer. This avoids the error changing by accident with `--bless`, you'd have to make an intentional decision to change the error message.